### PR TITLE
Allow multiple waypoints

### PR DESCRIPTION
**Pull request by boldtrn**
This PR allows to convert requests with multiple waypoints to be converted. I tested the change with the Mapbox SDK and everything seems to work as expected.

Sometimes the Mapbox SDK seems to behave a bit odd, when using multiple waypoints (like skipping one). But the same happens with the Mapbox API, so this is not an issue of this PR.

### CODE REVIEW

File: mapbox/src/main/java/com/graphhopper/navigation/mapbox/MapboxResponseConverter.java [hunk 1]
```diff
@@ -50,31 +50,35 @@ public static ObjectNode convertFromGHResponse(GHResponse ghResponse, Translatio
...
             for (int j = 0; j < instructions.size(); j++) {
                 ObjectNode instructionJson = steps.addObject();
-                putInstruction(instructions, j, locale, translationMap, instructionJson);
+                putInstruction(instructions, j, locale, translationMap, instructionJson, isFirstInstructionOfLeg);
+                Instruction instruction = instructions.get(j);
+                time += instruction.getTime();
+                distance += instruction.getDistance();
+                isFirstInstructionOfLeg = false;
+                if (instruction.getSign() == Instruction.REACHED_VIA || instruction.getSign() == Instruction.FINISH) {
+                    putLegInformation(legJson, path, i, time, distance);
```
Comment (karussell) [inline mapbox/src/main/java/com/graphhopper/navigation/mapbox/MapboxResponseConverter.java]: After this, should the time and distance be set to 0?

Comment (boldtrn) [reply mapbox/src/main/java/com/graphhopper/navigation/mapbox/MapboxResponseConverter.java]: Ah yes, thanks for catching this. Will run a new set of tests and check if everything works!

**Comment by boldtrn** [comment]
I tested the updated version with the time and duration reset. Everything looks good to go. Let's merge this PR :)

**Comment by karussell** [comment]
:+1:

**Pull request merged**